log/Logger.js

```diff
@@ -1,26 +1,48 @@
-const moment = require('moment')
-const winston = require('winston')
+const util = require('util');
+const {createLogger, format, transports} = require('winston');
+const {errors, json, metadata, timestamp} = format;
 
-const container = new winston.Container()
-
-function timestamp() {
-  return moment().format('YYYY-MM-DDTHH:mm:ssZ')
-}
-
-function transport(label) {
-  return new winston.transports.Console({
-    level: (process.env.LOG_LEVEL || 'INFO').toLowerCase(),
-    json: process.env.JSON_PRINT || false,
-    timestamp: timestamp,
-    label: label
-  })
-}
+const customMessageFormat = format((info) => {
+  const {message, metadata} = info;
+  if (metadata && typeof metadata === 'object' && Object.keys(metadata).length) {
+    info.message = message + util.inspect(metadata);
+  }
+  // delete metadata as its concatenated to the message
+  delete info.metadata;
 
-class Logger {
+  return info;
+});
 
-  static getLogger(name) {
-    return container.add(name, {transports: [transport(name)]})
-  }
-}
+/**
```

Comment:
We can add/remove fields as required. Currently, I have come up with this structure
```
{
    "level":"info",
    "timestamp":"2021-01-14T11:40:54.859Z",
    "message":"Hello World { hello: 'world' }",
    "moduleName":"loggerTest",
    "stack": "Error: ...."
 }
 ```
 Do we need more fields?